manifests/init.pp

```diff
@@ -9,7 +9,8 @@
 # @param bindir Where to create symlinks for our commands
 # @param libdir The directory where plugins will go in
 # @param configdir Root directory to config files
-# @param facts_refresh_interval Minutes between fact refreshes, set to 0 to disable cron based refreshes
+# @param facts_refresh_interval Minutes between fact refreshes, set to 0 to disable periodic based refreshes
+# @param periodic_runmode Defaults to "cron" and may be set to "systemd.timer"
```

Comment:
The data type and default value in the code make it possible to focus on what this setting is for:

```suggestion
# @param periodic_runmode Specify which provider to use for periodic actions
```